GliaWidgets/Sources/CallVisualizer/ScreenSharing/Model/ScreenSharingViewModel.swift

```diff
@@ -0,0 +1,109 @@
+import SwiftUI
+import Combine
+
+extension CallVisualizer.ScreenSharingView {
+    class Model: ObservableObject {
```

Comment:
would we inherit some classes from this model? if not, let's make it `final`.

Reply:
fixed [2f45a67](https://github.com/salemove/ios-sdk-widgets/pull/784/commits/2f45a67ef85d8be264e9fdf761f0925da56dfc7b)